devito/core/autotuning.py

```diff
@@ -89,9 +89,15 @@ def autotune(operator, args, level, mode):
 
     # Perform autotuning
     timings = {}
+    processed = []
```

Comment:
nitpicking: I'd call it "seen" for homogeneity with rest of codebase, and I'd turn it into a set.